blog/2023-11-17-hacktoberfest2/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Uptane's Successful Venture in Hacktoberfest: Building Community and Polishing Our New Website 🎃"
+authors:
+  - Phil
+--- 
+
+## Introduction
+Uptane, the pioneering software update security system for the automotive industry, recently took a significant leap in its community engagement efforts. Following the launch of our revamped website, a project that was part of the Google Summer of Code, we decided to participate in Hacktoberfest. This decision, suggested by one of our core website contributors, [Abhijay Jain](https://github.com/Abhijay007), marked a strategic step to leverage our recent advancements and to further solidify our presence in the open-source community. Our initial announcement about participating in Hacktoberfest can be found in our [previous blog post](https://uptane.org/blog/2023/10/28/hacktoberfest).
+
+## Participation in Hacktoberfest: Goals and Strategy
+Our primary goal for participating in Hacktoberfest was twofold: to grow our vibrant open-source community and to address the lingering issues from our website's transition. With a new site comes new challenges, and we saw Hacktoberfest as an ideal platform to engage with skilled contributors who could help us refine and polish our online presence.
+
+## Community Engagement: Discord, GitHub, and Beyond
+Community engagement was at the heart of our Hacktoberfest strategy. Our core community members played a crucial role in creating and managing issues, reviewing pull requests, and facilitating meaningful interactions with contributors. We leveraged Discord for real-time communication and discussions, while GitHub's robust communication tools were instrumental in managing our project workflows. This integrated approach allowed for efficient and effective collaboration, making the most of the talents and enthusiasm of the open-source community.
+
+## Contributions: Diverse and Impactful
+During Hacktoberfest, Uptane received a variety of contributions that significantly enhanced our project. We were thrilled to see 14 pull requests, each addressing a separate issue. These ranged from textual improvements to our website to the technical advancements of setting up automated Continuous Integration (CI) pipeline workflows and automations. This diversity in contributions reflects the breadth of skills and dedication within our community. A detailed list of the resolved issues can be found [here](https://github.com/uptane/uptane.github.io/issues?q=is%3Aissue+is%3Aclosed+label%3Ahacktoberfest). Special thanks to our Hacktoberfest contributors, [Tejas Raskar](https://github.com/tejas-raskar), [Shubham Sharma](https://github.com/shubhusion), [Ashmit JaiSarita Gupta](https://github.com/devilkiller-ag), [Mik Watkins](https://github.com/Mikerniker), [Anuj Mittal](https://github.com/mit-anuj), [Ojas Mittal](https://github.com/AmosBlack), and [Abhijay Jain](https://github.com/Abhijay007)!
```

Comment:
Do we need "automation/automated twice in the third sentence. Couldn't we just end the sentence after "pipeline"? This makes it sound like other automated elements were added. If this is the case then remove the "automated" before CI and let the sentence read "...setting up a Continuous Integration pipeline and other automated tasks."

Reply:
> Do we need "automation/automated twice in the third sentence. Couldn't we just end the sentence after "pipeline"? This makes it sound like other automated elements were added. If this is the case then remove the "automated" before CI and let the sentence read "...setting up a Continuous Integration pipeline and other automated tasks."

@jhdalek55 You are correct. This sounds awkward. I changed the sentence "These ranged from simple textual improvements to our website to the setting up advanced Continuous Integration (CI) workflows and automations for our repository."